lib/src/main/java/org/team100/lib/util/CameraAngles.java

```diff
@@ -1,47 +1,28 @@
 package org.team100.lib.util;
 
 public class CameraAngles {
-    private final double m_downwardAngleDegrees;
-    private final double m_horzFOVDegrees;
-    private final double m_vertFOVDegrees;
-    private final double m_horzResolution;
-    private final double m_vertResolution;
+    private final double m_downwardAngleRads;
+    private final double m_horzAngle = 0;
```

Comment:
seems like this isn't used?

we have a way to remember camera offsets (see the Camera class), so you should use that instead of making another one here.